spec/mailers/previews/placements/school_user_mailer_preview.rb

```diff
@@ -34,6 +50,25 @@ def user
     )
   end
 
+  def school_with_appetite(appetite, year)
+    school = Placements::School.new(
+      id: stubbed_id,
+      name: "Test School",
+    )
+
+    hosting = Placements::HostingInterest.new(
```

Comment:
```suggestion
    hosting_interest = Placements::HostingInterest.new(
```
Could we make this more descriptive?